### PR TITLE
[MLIR] Enable nonxdlops conv fwd tests for int8

### PR DESCRIPTION
**Pull request by jerryyin**
Follow-up to https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1539

After supports for xdlops/nonxdlops for int8 added, only xdlops testing has been added for the previous PR. I'm adding the non-xdlops testing as well such that it get tracked in MIOpen CI. Now both forward test has `INT8_ENABLED` flags in them.

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -693,7 +693,7 @@ if ((NOT CMAKE_BUILD_TYPE MATCHES "DEBUG") OR (NOT ${CODECOV_TEST}))
     )
 endif()
 
-add_custom_test(test_conv_igemm_mlir SKIP_UNLESS_ALL HALF_ENABLED SKIP_UNLESS_MLIR GFX900_DISABLED GFX908_DISABLED GFX90A_DISABLED
+add_custom_test(test_conv_igemm_mlir_fw SKIP_UNLESS_ALL HALF_ENABLED INT8_ENABLED SKIP_UNLESS_MLIR GFX900_DISABLED GFX908_DISABLED GFX90A_DISABLED
```
Comment (atamazov) [inline test/CMakeLists.txt:696]:
```suggestion
add_custom_test(test_conv_igemm_mlir_fwd SKIP_UNLESS_ALL HALF_ENABLED INT8_ENABLED SKIP_UNLESS_MLIR GFX900_DISABLED GFX908_DISABLED GFX90A_DISABLED
```

File: test/CMakeLists.txt [hunk 2]
```diff
@@ -702,7 +702,9 @@ add_custom_test(test_conv_igemm_mlir SKIP_UNLESS_ALL HALF_ENABLED SKIP_UNLESS_ML
     COMMAND ${IMPLICITGEMM_MLIR_ENV_F} $<TARGET_FILE:test_conv2d> ${IMPLICITGEMM_MLIR_ARGS_F} --input 128 64   56 56 --weights 64   64   1 1 --pads_strides_dilations 0 0 1 1 1 1
     COMMAND ${IMPLICITGEMM_MLIR_ENV_F} $<TARGET_FILE:test_conv2d> ${IMPLICITGEMM_MLIR_ARGS_F} --input 128 64   56 56 --weights 64   64   1 1 --pads_strides_dilations 0 0 1 1 1 1 --in_layout NHWC --fil_layout NHWC --out_layout NHWC
     COMMAND ${IMPLICITGEMM_MLIR_ENV_F} $<TARGET_FILE:test_conv2d> ${IMPLICITGEMM_MLIR_ARGS_F} --input 256 256  56 56 --weights 256  64   1 1 --pads_strides_dilations 0 0 1 1 1 1 --group-count 4
+)
 
+add_custom_test(test_conv_igemm_mlir_bw SKIP_UNLESS_ALL HALF_ENABLED SKIP_UNLESS_MLIR GFX900_DISABLED GFX908_DISABLED GFX90A_DISABLED
```
Comment (atamazov) [inline test/CMakeLists.txt:707]:
```suggestion
add_custom_test(test_conv_igemm_mlir_bwd SKIP_UNLESS_ALL HALF_ENABLED SKIP_UNLESS_MLIR GFX900_DISABLED GFX908_DISABLED GFX90A_DISABLED
```

**Review by atamazov — DISMISSED** [review]
This is good, but let's unify test names a bit.

File: test/CMakeLists.txt (hunk 1 above)
Comment (jerryyin) [reply test/CMakeLists.txt]: Done in 2b3e6c8

File: test/CMakeLists.txt (hunk 2 above)
Comment (jerryyin) [reply test/CMakeLists.txt]: Done in 2b3e6c8

**Review by atamazov — APPROVED** [review]
LGTM!

**Review by junliume — APPROVED** [review]

**Pull request merged**